_develop/connect/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: learn
+permalink: /:collection/:path.html
+---
+# Overview
+{:.no_toc}
+
+* TOC
+{:toc}
+
+Blockstack Connect is a Javascript library for integrating your application with Stacks v2. With Connect, you get some big benefits:
+
+- Pre-built UI to introduce your users to Blockstack, before they authenticate
+- Integration with our new authenticator
```

Comment:
what's the end benefit of that?

Reply:
Better conversion rates, higher education rates

Reply:
Would write that instead - "new authenticator" isn't necessarily appealing to someone who isn't familiar with the old one

Reply:
"Easily onboard new users and authenticate existing ones with plug-and-play Secret Key generation and username management"